app/uk/gov/hmrc/cgtpropertydisposals/service/returns/CgtCalculationService.scala

```diff
@@ -38,7 +38,8 @@ trait CgtCalculationService {
     reliefDetails: CompleteReliefDetailsAnswers,
     exemptionAndLosses: CompleteExemptionAndLossesAnswers,
     estimatedIncome: AmountInPence,
-    personalAllowance: AmountInPence
+    personalAllowance: AmountInPence,
+    initialGainOrLossCheck: Option[AmountInPenceWithSource]
```

Comment:
why is this a `check`? I'd just call is `initialGainOrLoss`

Reply:
renamed it to `initialGainOrLoss` and renamed the `val` we go on to instantiate inside to `initialGainOrLossWithSource`